Description: Include <sys/sysmacros.h> to properly define minor()
Origin: upstream, https://github.com/open-iscsi/open-iscsi/commit/6d68ef5871c94c6ebbbe6e6b1fe0bc2dce711052

Index: open-iscsi/iscsiuio/src/unix/libs/bnx2x.c
===================================================================
--- open-iscsi.orig/iscsiuio/src/unix/libs/bnx2x.c
+++ open-iscsi/iscsiuio/src/unix/libs/bnx2x.c
@@ -50,6 +50,7 @@
 #include <sys/user.h>
 #include <fcntl.h>
 #include <unistd.h>
+#include <sys/sysmacros.h>
 
 #include "config.h"
 
Index: open-iscsi/iscsiuio/src/unix/libs/bnx2.c
===================================================================
--- open-iscsi.orig/iscsiuio/src/unix/libs/bnx2.c
+++ open-iscsi/iscsiuio/src/unix/libs/bnx2.c
@@ -46,6 +46,7 @@
 #include <sys/user.h>
 #include <fcntl.h>
 #include <unistd.h>
+#include <sys/sysmacros.h>
 
 #include "config.h"
 
